docs/source/guides/build.rst

```diff
@@ -233,7 +214,8 @@ Build Process
 
 Cloning the Repository
```

Comment:
I'd move this block to environment preparation, somewhere before the `vcpkg` step, because it does not apply for
- docker users, because they must have cloned it to get the container
- all other users, because they cloned it in the `vcpkg` step.

Reply:
In new structure (that I've sent yesterday) the whole part will be in "Prepare" section